yoda.py

```diff
@@ -275,4 +275,7 @@ def weather(ctx, input):
             test_string += i + " "
         data = sys.modules["modules.weather"].get_weather(test_string)
     else:
-        click.echo("No input specified. Run with --help for info")
+        click.echo('No input specified. Run with --help for info')
+
+from modules import keep
```

Comment:
It would be better if you could move the import statement to the top of `yoda.py`.